### ISSUE TITLE
Allowed domains, but no blocked domains in Settings

### ISSUE THREAD

**Issue by ingoj** [friendica/friendica#8762]
Hi!

There is an allowed friend domains/allowed mail domains setting in Friendica, but no list of blocked domains/mail domains: 

<img width="812" alt="Bildschirmfoto 2020-06-12 um 11 02 37" src="https://user-images.githubusercontent.com/18173881/84485328-47efe080-ac9c-11ea-92df-a72970d59fa3.png">

I think it would make sense and be good to also have a list of blocked domains, both for federation sites to block as well as mail domains to block.

**Comment by MrPetovan** [comment]
Hi @ingoj , this feature has a dedicated page in the admin menu on the left of the very page you took a picture of. Module path is `/admin/blocklist/server`.

**Comment by ingoj** [comment]
Ah, indeed... ETOOMANYOPTIONS ;)

Then maybe it would be good to add a note to those settings and maybe a link or put both (Allowed/Blocked on the same page). 
Nothing that needs to be resolved now, though... :)

**Comment by ingoj** [comment]
@MrPetovan are you sure that /admin/blocklist/server is for blocking maildomains from registrations?

Reason is: lately I have many newly registered accounts that are being used for spamming, mainly from gmail.com domain, so I would like to prevent new accounts created with that maildomain...

**Comment by MrPetovan** [comment]
No, it's only for remote communication. I likely gave you the wrong information initially.

**Comment by loma-one** [comment]
Is there a way to block one or more domains from registering with Friendica? At the moment I have more and more registrations from Indian and US American SEDO posters, which I have to sort out manually. In some cases, they proceed quite cleverly. First they create an account, leave it dormant until they slip out of focus, and then fill it with links. a Work Arround would already help once. 

Optimal would be of course, if only addresses are sorted out, which use a number sequence in the alias.

**Comment by MrPetovan** [comment]
I guess we can filter the email address domain on the domain blocklist, but I'm not confortable using any filter on the username portion of it.

**Comment by loma-one** [comment]
Filtering the name is not so crucial. 
But it would be good if I can say before registration that certain domain names are not allowed and will be rejected.

The worst case is that I already have a user on the system and have to delete him. This is all the more unpleasant, since I have no way to provide a reason that will be brought to the user's attention. Deleting without comment is extremely unfriendly in my view.
